samples/snippets/webhook_prebuilt_telecom.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2021, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" handle_webhook will return the correct fullfilment response dependong the tag that is sent in the request"""
```

Comment:
Remove leading whitespace and end with a "." Also typo correction for "dependong" --> "depending no"

Reply:
Done